atomic_reactor/plugins/pre_koji.py

```diff
@@ -45,7 +45,17 @@ def run(self):
             self.log.error("provided target '%s' doesn't exist", self.target)
             raise RuntimeError("Provided target '%s' doesn't exist!" % self.target)
         tag_info = self.xmlrpc.getTag(target_info['build_tag_name'])
+
+        if not tag_info or 'name' not in tag_info.keys():
```

Comment:
No need to actually call `keys()`. The in operator will work as expected by checking the keys in a `dict`.